README.md

```diff
@@ -456,7 +457,6 @@ import pytz
 from datetime import datetime
 
 conn = trino.dbapi.connect(
-    experimental_python_types=True,
```

Comment:
this code part can be omitted.

Reply:
It is already omitted.

Reply:
I think Michiel meant this entire snippet can now be removed since it tries to demo Python types.

We can instead change this snippet to return a value which cannot be represented with Python types like a negative DATE and show how to use `legacy_primitive_types`.